index.d.ts

```diff
@@ -1,5 +1,26 @@
-class PolyK {
-    static Slice(p: number[][][], ax: number, ay: number, bx: number, by: number)
+declare var PolyK: {
+    Slice(p: number[], ax: number, ay: number, bx: number, by: number): number[],
+    IsSimple(p: number[]): boolean,
+    IsConvex(p: number[]): boolean,
+    GetArea(p: number[]): number,
+    GetAABB(p: number[]): {},
+    Triangulate(p: number[]): number[],
+    ContainsPoint(p: number[], ax: number, ay: number): boolean,
+    Raycast(p: number[],x : number, y: number, dx: number, dy: number): RaycastResult,
+    ClosestEdge(p: number[], x: number, y: number): RaycastResult
 }
-namespace PolyK {}
```

Comment:
The namespace was needed to export a class and be able to import the module.

We should add a typescript test file `test/types.ts`

```javascript
import * as PolyK from '../'

PolyK.ClosestEdge([1, 2], 1, 2)
...
```